modules/commons/src/main/resources/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.2.json

```diff
@@ -0,0 +1,164 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.1/shapes/"
```

Comment:
bump this version too to neurosciencegraph/commons/typedlabeledontologyterm/v0.1.2

Reply:
Done